stytch/b2b/sessions.go

```diff
@@ -16,18 +16,23 @@ import (
 	"github.com/golang-jwt/jwt/v5"
 	"github.com/mitchellh/mapstructure"
 	"github.com/stytchauth/stytch-go/v11/stytch"
+	"github.com/stytchauth/stytch-go/v11/stytch/b2b/rbac"
 	"github.com/stytchauth/stytch-go/v11/stytch/b2b/sessions"
+	"github.com/stytchauth/stytch-go/v11/stytch/shared"
 	"github.com/stytchauth/stytch-go/v11/stytch/stytcherror"
 )
 
 type SessionsClient struct {
-	C    stytch.Client
-	JWKS *keyfunc.JWKS
```

Comment:
Meta-thought: I don't _think_ the `XClient`s should be exposing any of these fields externally -- we should hide this and only expose the method calls and sub-services. With that said, that can go into the backlog for SDK improvements.